src/fontra/client/web-components/ui-form.js

```diff
@@ -163,9 +163,9 @@ export class Form extends SimpleElement {
       }
 
       let label = fieldItem.label || fieldItem.key || "";
-      if (label.length && fieldItem.type !== "header") {
+      /* if (label.length && fieldItem.type !== "header") {
         label += ":";
-      }
+      } */ // Conflicts with colons within localization values
```

Comment:
Can you elaborate on this problem?

Reply:
The main issue here is the difference of latin colon `:` and CJK colon `：`, they are different.
I'd like to use the correct colon for the corresponding language, thus I comment out this function to avoid appending a latin colon after Chinese phrases.
We can hardcode the colons in the language files instead.

Reply:
Thanks, understood!